src/v2/view-builder/ViewFactory.js

```diff
@@ -72,6 +72,9 @@ const VIEWS_MAPPING = {
   'success-redirect': {
     [DEFAULT]: SuccessView,
   },
+  'redirect': {
+    [DEFAULT]: RedirectView,
```

Comment:
Given it's identical to IdentifyView, maybe just point to it and add comments regarding why such 'hack' ?

Reply:
I did go back and forth on this one, but given that this view is very simple today we can use IdentifyView. I created this thinking I might be able to move some logic specific to this view here.

Reply:
I see. sure, we can refactor whenever needed.